Makefile

```diff
@@ -20,7 +20,7 @@ test_all_sql: test_postgres test_mysql
 .PHONY: test_postgres
 test_postgres: start_postgres_10 run_postgres_sql_tests stop_postgres_10 start_postgres_11 run_postgres_sql_tests stop_postgres_11 start_postgres_12 run_postgres_sql_tests stop_postgres_12 start_postgres_13 run_postgres_sql_tests stop_postgres_13
 .PHONY: test_mysql
-test_mysql: start_mysql_80 run_mysql_sql_tests stop_mysql_80 start_mysql_57 run_mysql_sql_tests stop_mysql_57
+test_mysql: start_mysql_80 run_mysql_sql_tests stop_mysql_80 run_mysql_sql_tests
```

Comment:
Shouldn't this be `start_mysql_80 run_mysql_sql_tests stop_mysql_80` ?

Reply:
good catch, we don't need to run the mysql tests twice now